### PR TITLE
Multi: Update wallet settings page to the latest UI

### LINKED ISSUE #987: V2 wallet settings update
The wallet settings page has been updated on the v2 layout design to implement single wallet selection.

https://www.figma.com/file/e0cYaAzkXyXXPY00v23cQY/GoDCR-%26-DCRMobile?node-id=3591%3A140470

### PR DESCRIPTION
**Pull request by devchoplife**
Closes #987, #1006
This PR updates the wallet settings page to the latest mockup.
Images:
image

image

image

image

image

### CODE REVIEW

File: ui/page/wallets/wallet_settings_page.go [hunk 1]
```diff
@@ -69,17 +85,38 @@ func (pg *WalletSettingsPage) Layout(gtx layout.Context) layout.Dimensions {
...
+					func(gtx C) D {
+						return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+							layout.Rigid(pg.debug()),
+						)
+					},
+					func(gtx C) D {
+						return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+							layout.Rigid(pg.dangerZone()),
+						)
+					},
```
Comment (Sirmorrison) [inline ui/page/wallets/wallet_settings_page.go]:
```suggestion
					func(gtx C) D {
								if !pg.wallet.IsWatchingOnlyWallet() {
									return pg.changePassphrase()(gtx)
								}
								return layout.Dimensions{}
					},
					pg.stakeshuffle(),
					pg.debug(),
					pg.dangerZone(),
```

File: ui/page/wallets/wallet_settings_page.go [hunk 2]
```diff
@@ -112,10 +183,63 @@ func (pg *WalletSettingsPage) debug() layout.Widget {
...
+	return func(gtx C) D {
+		return pg.pageSections(gtx, values.String(values.StrStakeShuffle),
+			func(gtx C) D {
+				return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						mixedAccountRow := row{
+							title:     values.String(values.StrMixedAccount),
+							clickable: pg.mixedAccount,
+							icon:      pg.chevronRightIcon,
+							label:     pg.Theme.Body2("mixed"), // TODO
```
Comment (Sirmorrison) [inline ui/page/wallets/wallet_settings_page.go]: localized string

File: ui/page/wallets/wallet_settings_page.go [hunk 3]
```diff
@@ -112,10 +183,63 @@ func (pg *WalletSettingsPage) debug() layout.Widget {
...
+						}
+						return pg.clickableRow(gtx, mixedAccountRow)
+					}),
+					layout.Rigid(pg.Theme.Separator().Layout),
+					layout.Rigid(func(gtx C) D {
+						changeAccountRow := row{
+							title:     values.String(values.StrChangeAccount),
+							clickable: pg.changeAccount,
+							icon:      pg.chevronRightIcon,
+							label:     pg.Theme.Body2("unmixed"), // TODO
```
Comment (Sirmorrison) [inline ui/page/wallets/wallet_settings_page.go]: localize string

File: ui/page/wallets/wallet_settings_page.go [hunk 4]
```diff
@@ -112,10 +183,63 @@ func (pg *WalletSettingsPage) debug() layout.Widget {
...
 func (pg *WalletSettingsPage) dangerZone() layout.Widget {
 	return func(gtx C) D {
 		return pg.pageSections(gtx, values.String(values.StrDangerZone),
-			pg.bottomSectionLabel(pg.deleteWallet, values.String(values.StrRemoveWallet)))
+			func(gtx C) D {
+				return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						return layout.Inset{Top: values.MarginPadding15, Bottom: values.MarginPadding10}.Layout(gtx, func(gtx C) D {
+							return layout.Flex{}.Layout(gtx,
+								layout.Rigid(pg.Theme.Label(values.TextSize16, "Allow spending from unmixed accounts").Layout),
```
Comment (Sirmorrison) [inline ui/page/wallets/wallet_settings_page.go]: localize string

File: ui/page/wallets/wallet_settings_page.go [hunk 5]
```diff
@@ -214,7 +338,7 @@ func (pg *WalletSettingsPage) HandleUserInteractions() {
 									m.SetLoading(false)
 									return
 								}
-								pg.Toast.Notify("Spending password updated")
+								pg.Toast.Notify("Spending passphrase updated")
```
Comment (Sirmorrison) [inline ui/page/wallets/wallet_settings_page.go]: localize string

File: ui/page/wallets/wallet_settings_page.go [hunk 6]
```diff
@@ -203,8 +327,8 @@ func (pg *WalletSettingsPage) HandleUserInteractions() {
 					newSpendingPasswordModal := modal.NewCreatePasswordModal(pg.Load).
 						Title(values.String(values.StrChangeSpendingPass)).
 						EnableName(false).
-						PasswordHint("New spending password").
-						ConfirmPasswordHint("Confirm new spending password").
+						PasswordHint("New spending passphrase").
+						ConfirmPasswordHint("Confirm new spending passphrase").
```
Comment (Sirmorrison) [inline ui/page/wallets/wallet_settings_page.go]: localize string

**Review by Sirmorrison — CHANGES_REQUESTED** [review]

**Comment by JustinBeBoy** [comment]
App crash when cancel change passphrase

image

**Comment by devchoplife** [comment]
> App crash when cancel change passphrase
> 
> image

I created an issue here #1006

**Review by JustinBeBoy — DISMISSED** [review]

File: ui/values/localizable/en.go [hunk 7]
```diff
@@ -109,14 +109,14 @@ unlockWithPassword = "Unlock with password"
...
 "confirmSpendingPassword" = "Confirm spending password";
-"changeSpendingPass" = "Change spending password";
+"changeSpendingPass" = "Change spending passphrase";
 "currentSpendingPassword" = "Current spending password";
-"newSpendingPassword" = "New spending password";
-"confirmNewSpendingPassword" = "Confirm new spending password";
-"spendingPasswordUpdated" = "Spending password updated";
+"newSpendingPassword" = "New spending passphrase";
+"confirmNewSpendingPassword" = "Confirm new spending passphrase";
+"spendingPasswordUpdated" = "Spending passphrase updated";
```
Comment (Sirmorrison) [inline ui/values/localizable/en.go]: Revert please.. it should be `Password`

**Review by Sirmorrison — CHANGES_REQUESTED** [review]

File: ui/page/wallets/wallet_settings_page.go [hunk 8]
```diff
@@ -98,6 +119,40 @@ func (pg *WalletSettingsPage) layoutMobile(gtx layout.Context, body layout.Widge
...
+			}),
+		)
+	})
+}
+
+func (pg *WalletSettingsPage) subSectionLabel(title string) layout.Widget {
+	return func(gtx C) D {
+		return pg.Theme.Body1(title).Layout(gtx)
+	}
+}
```
Comment (Sirmorrison) [inline ui/page/wallets/wallet_settings_page.go]: This is only used once.. we can remove.

File: ui/page/wallets/wallet_settings_page.go [hunk 9]
```diff
@@ -98,6 +119,40 @@ func (pg *WalletSettingsPage) layoutMobile(gtx layout.Context, body layout.Widge
...
+func (pg *WalletSettingsPage) subSection(gtx C, title string, body layout.Widget) D {
+	return layout.Inset{Top: values.MarginPadding5, Bottom: values.MarginPadding15}.Layout(gtx, func(gtx C) D {
+		return layout.Flex{}.Layout(gtx,
+			layout.Rigid(pg.subSectionLabel(title)),
+			layout.Flexed(1, func(gtx C) D {
+				return layout.E.Layout(gtx, body)
+			}),
+		)
+	})
+}
```
Comment (Sirmorrison) [inline ui/page/wallets/wallet_settings_page.go]:
```suggestion
func (pg *WalletSettingsPage) subSection(gtx C, title string, body layout.Widget) D {
	return layout.Inset{Top: values.MarginPadding5, Bottom: values.MarginPadding15}.Layout(gtx, func(gtx C) D {
		return layout.Flex{}.Layout(gtx,
			layout.Rigid(pg.Theme.Body1(title).Layout),
			layout.Flexed(1, func(gtx C) D {
				return layout.E.Layout(gtx, body)
			}),
		)
	})
}
```

**Review by Sirmorrison — CHANGES_REQUESTED** [review]
I noticed that to access the wallet setting page, you have to go through the wallets page.

According to the design, the wallet settings page should be accessed from the more page >> settings option.

image

The wallet page as been deprecated in #1008 however, till this PR is merged, leave the design as it is.. Just move wallet setting to more >>> settings.

File: ui/page/info/wallet_settings_page.go [hunk 10]
```diff
@@ -98,6 +119,40 @@ func (pg *WalletSettingsPage) layoutMobile(gtx layout.Context, body layout.Widge
...
+			}),
+		)
+	})
+}
+
+func (pg *WalletSettingsPage) subSectionLabel(title string) layout.Widget {
+	return func(gtx C) D {
+		return pg.Theme.Body1(title).Layout(gtx)
+	}
+}
```
Comment (Sirmorrison) [inline ui/page/info/wallet_settings_page.go]:
```suggestion
func (pg *WalletSettingsPage) subSection(gtx C, title string, body layout.Widget) D {
	return layout.Inset{Top: values.MarginPadding5, Bottom: values.MarginPadding15}.Layout(gtx, func(gtx C) D {
		return layout.Flex{}.Layout(gtx,
			layout.Rigid(pg.Theme.Body1(title).Layout),
			layout.Flexed(1, func(gtx C) D {
				return layout.E.Layout(gtx, body)
			}),
		)
	})
}
```

this `subSectionLabel` was only used once.

**Review by Sirmorrison — CHANGES_REQUESTED** [review]
This was the review request 
image

you dont need the wallet setting option as added.
the settings is now the new wallet setting.. meaning clicking on the `settings` option should open the `wallet settings page`

image

Also, move wallet_settings_page.go file from the `info` package to the root directory `page`

**Comment by Sirmorrison** [comment]
Ok.. the PR looks good so far.

Since we are still on the wallet settings page.. kindly update the UI including the following sections.

image

**Review by Sirmorrison — APPROVED** [review]

**Pull request merged**